<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpScannerDialog</class>
 <widget class="QWidget" name="IpScannerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP Scanner</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="startIpLineEdit">
     <property name="placeholderText">
      <string>Start IP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="endIpLineEdit">
     <property name="placeholderText">
      <string>End IP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="findMyIpButton">
     <property name="text">
      <string>Find My IP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="scanButton">
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save to HTML</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="filterUnreachableRadioButton">
     <property name="text">
      <string>Filter Unreachable Addresses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="outputTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
